src/dxfdim.cc

```diff
@@ -60,13 +61,24 @@ ValuePtr builtin_dxf_dim(const Context *ctx, const EvalContext *evalctx)
 		ValuePtr n = evalctx->getArgName(i);
 		ValuePtr v = evalctx->getArgValue(i);
 		if (evalctx->getArgName(i) == "file") {
-			filename = lookup_file(v->toString(), 
+			rawFilename = v->toString();
+			filename = lookup_file(rawFilename, 
 			evalctx->documentPath(), ctx->documentPath());
```

Comment:
add warning, when a named parameter is supplied more then once (applies to other places here as well)

Reply:
I move that to an other to do list